docs/UserGuide.md

```diff
@@ -112,11 +112,78 @@ Examples:
 
 ## 1.5 Sort
 
-### 1.5.1 Sorting patients
+### 1.5.1 Sorting patients `sortpatient` 
 
-### 1.5.2 Sorting appointments
+Sorts patients by a single field
 
-### 1.5.3 Sorting bills
+Format:
+```sortpatient <prefix><input> ...```
+
+* The command word is `sortpatient`.
+* The prefixes are c/ for Criteria and o/ for Order.
+* The criteria can be Name of patient (name), Phone number of patient (phone), Email address of patient (email), Address of patient (address).
+* The order can be Ascending (asc) or Descending (desc).
+* If there are no prefixes keyed in, an error message will be shown with the correct command format.
+* If the input after a prefix is empty/invalid, an error message with the constraint of the field will be shown.
+
+Examples:
+* ```sortpatient c/name o/asc``` returns patients sorted by name in ascending order.
+
+<img src="images/sortpatient1.png" width="800px" height ="400px">
+
+* ```sortpatient c/phone o/desc``` returns patients sorted by phone number in descending order.
+
+<img src="images/sortpatient2.png" width="800px" height ="400px">
+
+### 1.5.2 Sorting appointments `sortappointment`
+
+Sorts appointments by a single field
+
+Format:
+```sortappointment <prefix><input> ...```
+
+* The command word is `sortappointment`.
+* The prefixes are c/ for Criteria and o/ for Order.
+* The criteria can be Name of patient (name), Medical Test of appointment (test), Slot of appointment (slot) and Doctor of appointment (doctor).
+* The order can be Ascending (asc) or Descending (desc).
+* If there are no prefixes keyed in, an error message will be shown with the correct command format.
+* If the input after a prefix is empty/invalid, an error message with the constraint of the field will be shown.
```

Comment:
again don't need this

Reply:
talked about this. More information is good